packages/nuxt/src/core/app.ts

```diff
@@ -117,6 +117,11 @@ export async function resolveApp (nuxt: Nuxt, app: NuxtApp) {
     const layoutFiles = await resolveFiles(config.srcDir, `${layoutDir}/**/*{${nuxt.options.extensions.join(',')}}`)
     for (const file of layoutFiles) {
       const name = getNameFromPath(file, resolve(config.srcDir, layoutDir))
+      if (!name) {
```

Comment:
Should we do the same for `middleware` ([here](https://github.com/nuxt/nuxt/blob/4dd87f6e281bacacafa9566a21388d5a7a53f911/packages/nuxt/src/core/app.ts#L130-L131))?

Reply:
good catch 👌

Reply:
(Note for anyone else looking that plugins + middleware are not resolved in a nested fashion.)